src-docs/src/components/guide_theme_selector/guide_theme_selector.tsx

```diff
@@ -45,8 +42,6 @@ export const GuideThemeSelector: React.FunctionComponent<GuideThemeSelectorProps
 // @ts-ignore Context has no type
 const GuideThemeSelectorComponent: React.FunctionComponent<GuideThemeSelectorProps> = ({
   context,
-  onToggleLocale,
-  selectedLocale,
```

Comment:
If these props aren't being used anymore, can they be removed from the props interface?